android/src/main/kotlin/com/lahaus/iterable_flutter/IterableFlutterPlugin.kt

```diff
@@ -98,6 +103,27 @@ class IterableFlutterPlugin : FlutterPlugin, MethodCallHandler {
         })
   }
 
+  private fun notifyOpenPushNotification(){
```

Comment:
I think that naming here could be addressed like:
`notifyPushNotificationOpened`